static-site/content/blog/2025-03-31-annual-update-march-2025.md

```diff
@@ -0,0 +1,74 @@
+---
+author: "Trevor Bedford, Richard Neher"
+date: "2025-03-31"
+title: "Nextstrain Annual Update March 2025"
+sidebarTitle: "Annual Update March 2025"
+---
+
+We continue to follow a model of genomic surveillance in which our central conceit has two pillars:
+
+1. *Routine real-time genomic surveillance across a variety of circulating pathogens*
+
+2. *Rapid pivots to emerging public health threats*
+
+as we believe that proper tooling for real-time surveillance (1) enables rapid response when new threats emerge (2).
+
+# Recent activities (March 2024 to March 2025)
+
+## Pathogen-specific analyses
+
+We've continued our tracking of evolution of **SARS-CoV-2** at [/ncov](/ncov/gisaid/global/6m), with nine new clades through 2024 and into 2025. We've continued to estimate multinomial logistic regression (MLR) fitness models and display these inferences for clades and lineages at [/sars-cov-2/forecasts](/sars-cov-2/forecasts). We display these lineage fitnesses on the tree via a new ["MLR lineage fitness"](/ncov/gisaid/global/6m?c=mlr_lineage_fitness) coloring. We also revamped [default clade coloring](https://github.com/nextstrain/ncov/pull/1132) to better distinguish between circulating clades. Tracking of **seasonal influenza** at [/seasonal-flu](/seasonal-flu) proceeds in a similar fashion, with emphasis on keeping clade nomenclature updated.
+
+In response to the **H5N1 cattle outbreak**, [we set up an automated analysis](/blog/2024-06-18-h5n1-cattle-outbreak-analysis-and-resources) that sources data from SRA and GenBank. The USDA does not immediately upload complete genomes with US state and collection date metadata and instead uploads unannotated raw reads to SRA. The Andersen Lab has been assembling these reads into consensus genomes. We ingest and deduplicate across these data sources. More recent zoonoses of [genotype D1.1 into cattle](/avian-flu/h5n1-d1.1/genome) have also been automated and shared in a similar fashion. In addition, Moncla Lab has continued to keep pace with **H5N1 evolution in birds**, including a narrative showcasing the [evolution and spread of H5N1 in North America](/community/narratives/moncla-lab/nextstrain-narrative-hpai-north-america@main/HPAI-in-North-America).
```

Comment:
Maybe link [Andersen Lab](https://andersen-lab.com/) and [Moncala Lab](https://lmoncla.github.io/monclalab/)?

Reply:
Resolved in 3576ccbf888df9289f183b379e003f85d1a389fe